### PR TITLE
Add a cdktf python Azure example

### PR DESCRIPTION
**Pull request by judy3**
Add a cdktf python Azure example, thank you!

### CODE REVIEW

File: examples/python/azure/Pipfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+cdktf = "~=0.0.16"
```
Comment (skorfmann) [inline examples/python/azure/Pipfile:10]: Please reference from [dist](https://github.com/hashicorp/terraform-cdk/blob/76da576abd9223861615144e5fe278c483850ea7/examples/python/docker/Pipfile#L10)

File: examples/python/azure/cdktf.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "language": "python",
+  "app": "pipenv run ./main.py",
+  "terraformProviders": ["aws@~> 2.0", "azurerm@~> 2.0.0"],
```
Comment (skorfmann) [inline examples/python/azure/cdktf.json:4]: The `aws` provider can be dropped from that list.

**Review by skorfmann — CHANGES_REQUESTED** [review]
Hey Judy, thanks for your contribution!

Besides from the inline comments, two remarks:

Please gitignore the `imports` directory. While it could be useful to add it under version control for a project, in the case of examples it would add up quite quickly.

Also, please make sure to sign the CLA as mentioned in [this comment](https://github.com/hashicorp/terraform-cdk/pull/350#issuecomment-681324365)

File: examples/python/azure/Pipfile (hunk 1 above)
Comment (judy3) [reply examples/python/azure/Pipfile]: Done.

File: examples/python/azure/cdktf.json (hunk 2 above)
Comment (judy3) [reply examples/python/azure/cdktf.json]: Done.

**Comment by judy3** [comment]
> Hey @judy3 - thanks for your contribution!
> 
> Besides from the inline comments, two remarks:
> 
> Please gitignore the `imports` directory. While it could be useful to add it under version control for a project, in the case of examples it would add up quite quickly.
> 
> Also, please make sure to sign the CLA as mentioned in [this comment](https://github.com/hashicorp/terraform-cdk/pull/350#issuecomment-681324365)

Hi skorfmann, I've signed the CLA, but not sure why the status doesn't change. 
image

**Comment by skorfmann** [comment]
> Hi @skorfmann, I've signed the CLA, but not sure why the status doesn't change. You can see the screenshot below:
> image

Hmm, can you help here @anubhavmishra?

**Comment by skorfmann** [comment]
While talking about this with @anubhavmishra, he pointed out that there's an error message in the @hashicorp-cla message:

> **Ubuntu** seems not to be a GitHub user.
> You need a GitHub account to be able to sign the CLA. If you already have a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

[Screenshot 2020-08-31 at 22 22 04](https://github.com/hashicorp/terraform-cdk/pull/350/commits/c415043f1f3a051133891a9673ebcf87bad9ec23)

@judy3 it looks like there's some default git config on your local machine, which should be changed. You can change the [git config](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) to match the email address of your Github profile. Besides from setting this for future commits, you'll have to [rewrite the commits](https://makandracards.com/makandra/1717-git-change-author-of-a-commit) for this pull request. 

Please let me know if there's anything unclear, I'm happy to help.

**Comment by judy3** [comment]
> While talking about this with @anubhavmishra, he pointed out that there's an error message in the @hashicorp-cla message:
> 
> > **Ubuntu** seems not to be a GitHub user.
> > You need a GitHub account to be able to sign the CLA. If you already have a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> 
> [Screenshot 2020-08-31 at 22 22 04](https://github.com/hashicorp/terraform-cdk/pull/350/commits/c415043f1f3a051133891a9673ebcf87bad9ec23)
> 
> @judy3 it looks like there's some default git config on your local machine, which should be changed. You can change the [git config](https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address) to match the email address of your Github profile. Besides from setting this for future commits, you'll have to [rewrite the commits](https://makandracards.com/makandra/1717-git-change-author-of-a-commit) for this pull request.
> 
> Please let me know if there's anything unclear, I'm happy to help.

Hi @skorfmann , I've deleted the previous commits and pushed again with correct git configuration on my local machine. Now it should fine. Thank you!

**Comment by skorfmann** [comment]
> Hi @skorfmann , I've deleted the previous commits and pushed again with correct git configuration on my local machine. Now it should fine. Thank you!

@judy3 Thanks!

**Review by skorfmann — APPROVED** [review]
Looks good 👍

**Pull request merged**